### PR TITLE
Add collection label next to search results

### PR DESCRIPTION
**Pull request by solpark**
**- Summary**
Closes issue #945 
Collection name is indicated on each search result item.

**- Test plan**
Shows collection name in each search result:
netlify-search-results

Does not show collection name next to items when not searched:
netlify-non-search-results

**- Description for the changelog**

Add collection label next to search results

**- A picture of a cute animal (not mandatory but encouraged)**
image

src: http://i.dailymail.co.uk/i/pix/2012/09/17/article-2204494-150E707A000005DC-700_634x438.jpg

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 02e0bf7a25ef615fae33d615473371cca92d0391

https://deploy-preview-1068--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 02e0bf7a25ef615fae33d615473371cca92d0391

https://deploy-preview-1068--cms-demo.netlify.com

**Comment by erquhart** [comment]
Thanks @solpark!

I'll give a short review each for code and design.

**Code**
We can simplify things by drilling down to the abstract goal: whenever entries from multiple collections are displayed at once, regardless of why, we should label each entry with it's collection title. The `EntryListing` component has a [`renderCardsForMultipleCollections`](https://github.com/netlify/netlify-cms/blob/756d562c6651b1634143f1f7d47ffc8119fe59bb/src/components/Collection/Entries/EntryListing.js#L43-L52) method that handles this specific case, so we can start there without altering components that are higher up in the tree.

That method wraps the `EntryCard` component directly, so we can simply pass in a `label` string when we want a label, and don't use the `label` prop when we don't. This approach should allow us to get away with only modifying two components total.

**Design**

The approach totally works, but I happen to know that there will eventually labels or tags or who-knows-what in that right side area of the entry card, so it might not be ideal placement. There's also precedent for placing a UI element's label in the top left, such as in editor fields and editorial workflow cards, both pictured below:

screen shot 2018-02-02 at 4 19 21 pm

screen shot 2018-02-02 at 3 59 05 pm

That last example is very similar to the card you're modifying - you can probably borrow those styles directly and maybe avoid writing new styles. Also like the last example, you can just use the collection name without prepending "Collection:".

Thanks again for taking the time to contribute! I hope the review was helpful - feel free to ask questions here or in [Gitter](https://gitter.im/netlify/netlifycms).

**Comment by solpark** [comment]
@erquhart  Thanks for the thorough feedback! I will make the changes ASAP :)

**Comment by solpark** [comment]
After changes: 
netlify-pr-change

**Comment by solpark** [comment]
@erquhart 
Hi! I made the changes but the netlify deploy-preview test is failing. I asked about this in Glitter and it seems to be a netlify issue. How should I go forward with this?

File: .all-contributorsrc [hunk 1]
```diff
@@ -709,6 +709,24 @@
...
+      "login": "solpark",
+      "name": "Sol Park",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/30510616?v=4",
+      "profile": "https://github.com/solpark",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "michaelromani",
```
Comment (tech4him1) [inline .all-contributorsrc]: @solpark Can you confirm that adding "Michael Romani" is intentional?

Comment (solpark) [reply .all-contributorsrc]: Yes, worked on this together!

**Comment by solpark** [comment]
Ah gotcha. Thank you!

**Comment by erquhart** [comment]
Did a bit of editing, borrowed styles from the workflow cards:

screen shot 2018-02-07 at 4 29 50 pm

**Comment by erquhart** [comment]
This is good to merge - thanks for taking it on @solpark!

**Pull request merged**